tests/loss/test_DINOLoss.py

```diff
@@ -0,0 +1,260 @@
+import itertools
+import unittest
+
+import numpy as np
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from lightly.loss import dino_loss
+from lightly.loss import DINOLoss
+
+
+class FacebookDINOLoss(nn.Module):
```

Comment:
maybe add a comment that you copy pasted this from somewhere so it's very clear what the purpose is